README.md

```diff
@@ -241,15 +277,17 @@ Suite Flags:
 
 
 Global Flags:
-      --cache-dir string   Directory to save the pricing and instance type caches (default "~/.ec2-instance-selector/")
-      --cache-ttl int      Cache TTLs in hours for pricing and instance type caches. Setting the cache to 0 will turn off caching and cleanup any on-disk caches. (default 168)
-  -h, --help               Help
-      --max-results int    The maximum number of instance types that match your criteria to return (default 20)
-  -o, --output string      Specify the output format (table, table-wide, one-line)
-      --profile string     AWS CLI profile to use for credentials and config
-  -r, --region string      AWS Region to use for API requests (NOTE: if not passed in, uses AWS SDK default precedence)
-  -v, --verbose            Verbose - will print out full instance specs
-      --version            Prints CLI version
+      --cache-dir string        Directory to save the pricing and instance type caches (default "~/.ec2-instance-selector/")
+      --cache-ttl int           Cache TTLs in hours for pricing and instance type caches. Setting the cache to 0 will turn off caching and cleanup any on-disk caches. (default 168)
+  -h, --help                    Help
+      --max-results int         The maximum number of instance types that match your criteria to return (default 20)
+  -o, --output string           Specify the output format (table, table-wide, one-line)
+      --profile string          AWS CLI profile to use for credentials and config
+  -r, --region string           AWS Region to use for API requests (NOTE: if not passed in, uses AWS SDK default precedence)
+      --sort-by string          Specify the field to sort by. Any quantity flag present in this CLI (vcpus, memory, gpu-memory-total, network-interfaces, spot-price, on-demand-price, instance-storage, ebs-optimized-baseline-bandwidth, ebs-optimized-baseline-throughput, ebs-optimized-baseline-iops, gpus, inference-accelerators) or a JSON path to the appropriate field in the instancetype.Details struct is acceptable. (default "$.InstanceType")
```

Comment:
Can you shorten the list to 2 examples. The flag description should ideally be pretty short.
Also, referencing a struct in the CLI flag description is a little awkward. The people using the tool aren't likely familiar with the internals of the tool. The output shape we use is almost exactly the response from `aws ec2 describe-instance-types`. It's probably worth adding an example output (in json format) of instancetype.Details below using the CLI tool w/ the --verbose option. 
```
      --sort-by string          Specify the field to sort by. Any quantity flag present in this CLI (vcpus, memory, etc) or a JSON path to the appropriate field in the instanceType.Details output shape is acceptable. (default ".InstanceType")
```

Reply:
I do agree that the current flag description is way too long, however, I feel as though it is important to notify users of what shorthand flags are available since not all CLI flags are usable as sorting fields. Also I'm unsure about how to mention the fact that json pathing is accepted without mentioning the struct. What do you think about moving such comments about the available flags and the use of json pathing to the sorting examples in the readme? They could exist as small notes after the example printout.

Reply:
sounds good to me!